README.md

```diff
@@ -1 +1,26 @@
-# community
\ No newline at end of file
+# Human Cell Atlas Data Coordination Platform community
+
+** Work in Progress **
+
+Welcome to the community!
+
+## Communicating
+
+To get started, [join the HumanCellAtlas Slack workspace](http://join-slack.humancellatlas.org/) to receive an invitation. 
+
+### Browse and join Slack channels:
+
+  - #general is for
+  - #dcp is for
+  - #ingestion-service is for
+  - #data-store is for
+  - #secondary analysis is for
+  - #data-portal is for
+  - ...
+
+## Community Governance
+### Oversight
+### Decision Making, Conflict Resolution, and Facilitation
+### Code of Conduct
+### Project ownership and the charter process
+### Enhancements and the RFC (Request for comments) process
```

Comment:
Has the RFC process been written up yet?  I see something basic here: https://github.com/HumanCellAtlas/RFCs  Will that SOP doc's home be here or in that repo?

Reply:
That repo was an early experiment by @sampierson which needs to be deleted. The community repository is the central location for all HCA DCP community governance/process.

A proposal for a RFC process can be found in [Appendix A](https://docs.google.com/presentation/d/1c2pfeck-b79Pjpo0-_E-2pKnePdX38XoDnezOryEFCo/edit#slide=id.g3c69396420_0_515) of the PM Retrospective slides.